plugin/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -9,7 +9,8 @@
     <cm:property-placeholder id="velocloudPluginProperties" persistent-id="org.opennms.velocloud" update-strategy="reload">
         <cm:default-properties>
             <cm:property name="url" value="http://127.0.0.1:8888/target" />
-            <cm:property name="apiKey" value="TOKEN" />
```

Comment:
We do not need these for now, as we have per requisition credentials.

Reply:
removed everything in the blueprint for now.